templates/bot/js/m365/README.md

```diff
@@ -1,33 +1,29 @@
+This is a simple search application with Messaging extension capabilities.
 
 ![Search App Demo](./images/SearchAppDemo.gif)
 
 ## Prerequisites
 
 - [NodeJS](https://nodejs.org/en/)
 - An M365 account. If you do not have M365 account, apply one from [M365 developer program](https://developer.microsoft.com/en-us/microsoft-365/dev-program)
-- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 1.55 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
+- [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version after 3.8.0 or [TeamsFx CLI](https://aka.ms/teamsfx-cli)
 
 ## Debug
 ### From Visual Studio Code
-- Use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Teams`, `Debug in Outlook` and click the `Run and Debug` green arrow button.
-- If you select `Debug in Outlook`, during debugging, a VS Code dialog will be popped up as the image below. Please click "Install in Teams" first to install the app in Teams, then click "Connect to Outlook channel" to connect the bot to Outlook channel, then click "Continue" to continue debugging the app in Outlook.
-
-  ![Install in Teams](./images/InstallInTeamsVSC.png)
+1. Use the `Run and Debug Activity Panel` in Visual Studio Code, select `Debug in Teams`, `Debug in Outlook` and click the `Run and Debug` green arrow button.
+2. If you select `Debug in Outlook`, during debugging, a VS Code dialog will be popped up as the image below. Please click "Install in Teams" first to install the app in Teams, then click "Configure Outlook" to connect the bot to Outlook channel, then click "Continue" to continue to debug the app in Outlook.
+  ![Install in Teams VSC Local](./images/InstallInTeamsVSCLocal.png)
 
 ### From TeamsFx CLI
-- From TeamsFx CLI: Start debugging the project by executing the command `teamsfx preview --local --m365-host <m365-host>` in your project directory, where `m365-host` is `teams` or `outlook`.
-- If you select `m365-host` as outlook, during debugging, a dialog will be popped up as the image below. Please select "Install in Teams" first to install the app in Teams, then select "Connect to Outlook channel" to connect the bot to Outlook channel, then select "Continue" to continue debugging the app in Outlook or Office.
-
-  ![Install in Teams](./images/InstallInTeamsCLI.png)
+1. From TeamsFx CLI: Start debugging the project by executing the command `teamsfx preview --local --m365-host <m365-host>` in your project directory, where `m365-host` is `teams` or `outlook`.
+2. If you select `m365-host` as `outlook`, during debugging, a dialog will be popped up as the image below. Please select "Install in Teams" first to install the app in Teams, then select "Configure Outlook" to connect the bot to Outlook channel, then select "Continue" to continue to debug the app in Outlook.
+  ![Install in Teams CLI Local](./images/InstallInTeamsCLILocal.png)
 
 ## Edit the manifest
 
```

Comment:
`The folder contains two manifest files` -> The folder contains a manifest template file.